### ISSUE TITLE
Drag/Drop File Uploading

### ISSUE THREAD

**Issue by bkonia** [formtools/core#507]
It would be nice if your file uploader supported drag/drop uploading, in addition to the old fashioned method of clicking a button, then selecting files.

**Comment by nelsondcosta23** [comment]
thats a nice ideia, FT3 already support multiupload

**Comment by benkeen** [comment]
Yeah, great idea. Ironically I just finished implementing that exact feature in my day job (boy IE11 is awful!). Will definitely keep this in mind.
